homeassistant/components/sungrow/translations/de.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Please remove the translations folder. All translation strings should go into `strings.json`